File: AppLovin-Android-Demo-App/src/main/java/com/applovin/apps/demoapp/banners/BannerDemoMenuActivity.java
package com.applovin.apps.demoapp.banners;

import android.content.Intent;

import com.applovin.apps.demoapp.DemoMenuActivity;
import com.applovin.apps.demoapp.DemoMenuItem;

public class BannerDemoMenuActivity
        extends DemoMenuActivity
{
    @Override
    protected DemoMenuItem[] getListViewContents()
    {
        DemoMenuItem[] result = {
                new DemoMenuItem( "Programmatic", "Programmatically creating an instance of it", new Intent( this, BannerProgrammaticActivity.class ) ),
                new DemoMenuItem( "Layout Editor", "Create a banner from the layout editor", new Intent( this, BannerLayoutEditorActivity.class ) ),
        };
        return result;
    }
}

File: AppLovin-Kotlin-Demo-App/src/main/java/com/applovin/apps/kotlindemoapp/banners/BannerDemoMenuActivity.kt
package com.applovin.apps.kotlindemoapp.banners

import android.content.Intent
import com.applovin.apps.kotlindemoapp.DemoMenuActivity
import com.applovin.apps.kotlindemoapp.DemoMenuItem

class BannerDemoMenuActivity : DemoMenuActivity()
{
    override fun getListViewContents(): Array<DemoMenuItem> = arrayOf(
            DemoMenuItem("Programmatic", "Programmatically creating an instance of it", Intent(this, BannerProgrammaticActivity::class.java)),
            DemoMenuItem("Layout Editor", "Create a banner from the layout editor", Intent(this, BannerLayoutEditorActivity::class.java))
    )
}
